test/runFIT.sh

```diff
@@ -148,6 +189,10 @@ main(){
     if [ ! -n "$TEST_LOG_LEVEL" ]; then
         TEST_LOG_LEVEL="4"
     fi
+
+    RACKHD_HOST=$(cat $RACKHD_TEST_DIR/config/stack_config.json |jq -r ".$TEST_STACK.rackhd_host")
+    RACKHD_HTTP_PORT=$(cat $RACKHD_TEST_DIR/config/stack_config.json |jq ".$TEST_STACK" |grep \"http\": |sed 's/[^0-9]*//g')
```

Comment:
can you make it more clear ?
RACKHD_HOST=$(           cat stack_config.json   |   jq ".$TEST_STACK.rackhd_host" )

 RACKHD_HTTP_PORT=$(cat stack_config.json   |   jq "$TEST_STACK.[\"install-config\"].ports.http"

Reply:
Thanks.
I already fixed the "RACKHD_HTTP_PORT"
But I don't change the "RACKHD_HOST" . Because the current directory is not "RackHD/test/config"